getting-started/cloudflare-workers.md

```diff
@@ -11,6 +11,7 @@ Let’s make your first application for Cloudflare Workers with Hono.
 
 A starter for Cloudflare Workers is available.
 Start your project with "create-hono" command.
+Select Cloudflare Workers template for this example
```

Comment:
I think it's better to write the actual output name instead of "Cloudflare Workers". Like the following:

```md
Select `cloudflare-workers` for this example.
```